protobuf/coreapi/api.proto

```diff
@@ -297,7 +297,8 @@ message DeployServiceReply {
 // }
 // ```
 message DeleteServiceRequest {
-  string serviceID = 1; // The Service ID. Generated when using the [`DeployService` API](#deployservice).
+  string serviceID = 1;  // The Service ID. Generated when using the [`DeployService` API](#deployservice).
+  bool deleteData = 2; // When enabled, any persistent data that belongs to service and its dependencies will be also deleted.
```

Comment:
```suggestion
  bool deleteData = 2; // When enabled, any persistent data (volumes) that belongs to service and its dependencies will be also deleted.
```

Reply:
How about _any volume_ to _any data volume_? Bu still, gRPC API is meant to be used by devs so they don't necessarily need to know about **volumes**. Keeping _persistent data_ might be  better.

Reply:
Here, i will say a gRPC user is more luckily to also be a service developer. In service with speak about volumes.

```suggestion
  bool deleteData = 2; // When enabled, any persistent data (volumes) that belongs to service and its dependencies will be also deleted.
```